### PR TITLE
167994447 Create database models

### PR DESCRIPTION
**Pull request by victor-shagor**
#### What does this PR do?
creates database models
#### Description of Task to be completed?
* create database models and migrations
#### How should this be manually tested?
* Clone the repo and `cd` into the project directory
* Run `yarn install` to install the project's dependency
* Setup environment variables
* Run `yarn migrate'

#### What are the relevant pivotal tracker stories?
[#167994447](https://www.pivotaltracker.com/story/show/167994447)

### CODE REVIEW

**Comment by minlinx** [comment]
Nice job, I think the **userId** column should be **owner** and a **foreign** **key**,  I don't think the **dev-start** script is that necessary, may be we should use **prestart** intead of **build** , check if the **build** folder exists, remove it before **babel** ...

File: src/models/User.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
     email: DataTypes.STRING,
     password: DataTypes.STRING,
-    id: { type: DataTypes.INTEGER, primaryKey: true }
-  }, {});
-  User.associate = function (models) {
-  };
+  }, {
+    classMethods: {
+      associate: (models) => {
+        // associations can be defined here
```
Comment (Daymorelah) [inline src/models/User.js]: Remove this comment.

File: src/models/accommodations.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    address: DataTypes.STRING,
+    roomName: DataTypes.STRING,
+    roomType: DataTypes.STRING,
+    vacantNumber: DataTypes.INTEGER,
+    image: DataTypes.STRING
+  }, {
+    classMethods: {
+      associate: (models) => {
+        // associations can be defined here
+        Accommodations.hasMany(models.User, {
```
Comment (Daymorelah) [inline src/models/accommodations.js:12]: Remove the comment on line 12

File: src/models/requests.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    userId: DataTypes.INTEGER,
+    origin: DataTypes.STRING,
+    destination: DataTypes.STRING,
+    flightDate: DataTypes.DATEONLY,
+    returnDate: DataTypes.DATEONLY,
+    AccomodationId: DataTypes.INTEGER
+  }, {
+    classMethods: {
+      associate: (models) => {
+        // associations can be defined here
```
Comment (Daymorelah) [inline src/models/requests.js]: Remove this comment.

**Review by Daymorelah — CHANGES_REQUESTED** [review]

File: src/config/config.js [hunk 4]
```diff
@@ -24,5 +24,4 @@ const config = {
   }
 };
 
-export default config;
-
+module.exports = config;
```
Comment (Daymorelah) [inline src/config/config.js:26]: remove the code on line 10 `operatorsAliases: false`

**Review by Daymorelah — CHANGES_REQUESTED** [review]

**Review by Daymorelah — COMMENTED** [review]
LGTM!

**Pull request merged**